unlock-app/src/components/content/event/Form.tsx

```diff
@@ -290,16 +314,27 @@ export const Form = ({ onSubmit }: FormProps) => {
                         },
                       })}
                       type="date"
+                      min={minEndDate}
                       label="End date"
                       error={
                         // @ts-expect-error Property 'event_start_date' does not exist on type 'FieldError | Merge<FieldError, FieldErrorsImpl<any>>'.
                         errors.metadata?.ticket?.event_end_date?.message || ''
                       }
                     />
                     <Input
-                      {...register('metadata.ticket.event_end_time')}
+                      {...register('metadata.ticket.event_end_time', {
+                        required: {
+                          value: true,
+                          message: 'This value is required',
+                        },
+                      })}
                       type="time"
+                      min={minEndTime}
                       label="End time"
+                      error={
+                        // @ts-expect-error Property 'event_end_time' does not exist on type 'FieldError | Merge<FieldError, FieldErrorsImpl<any>>'.
```

Comment:
You should be able to type this.

Reply:
is already defined in `NewEventForm` but this still gives ts error for nested parameters like `metadata.ticket.event_start_date`